### ISSUE TITLE
command fleet_status / Http Status Code 421 (MisdirectedRequest)

### ISSUE THREAD

**Issue by bassmaster187** [teslamotors/fleet-telemetry#180]
I receive sometime Http Statuscode 421 on command fleet_status. What does that mean?

**Comment by Bre77** [comment]
`This user is not present in the current region. See the regional requirements section for more information.`

https://developer.tesla.com/docs/fleet-api#response-codes

**Comment by bassmaster187** [comment]
Maybe related to this issue:
https://github.com/teslamotors/vehicle-command/issues/249

**Comment by Bre77** [comment]
> Maybe related to this issue:
> https://github.com/teslamotors/vehicle-command/issues/249
> 

To mitigate that I call `api/1/users/region` for every user login to ensure the region is correct. A month ago I had a handful of EU users showing up as NA, although that hasn't happened for a while.

**Comment by bassmaster187** [comment]
The Proxy Server should handle all the region stuff. At least for all other users.

**Comment by patrickdemers6** [comment]
I don't believe fleet_status raises 412 anymore. If you're still having issues, let me know. Also feel free to create an issue in vehicle-command if region-related logic would still be useful.
